client/src/components/ElementEditor/HoverBar.js

```diff
@@ -4,6 +4,45 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { inject } from 'lib/Injector';
 import { elementTypeType } from 'types/elementTypeType';
+import i18n from 'i18n';
+
+/**
+ * Render an hoverbar without any state
+ */
+function StatelessHoverBar({
+  AddElementPopoverComponent,
+  elementTypes,
+  elementId,
+  areaId,
+  popoverOpen,
+  onToggle }) {
+  const hoverBarClassNames = classNames('font-icon-plus-circled', 'hover-bar__line');
+  const areaClasses = classNames(
+    'hover-bar__area',
+    { 'hover-bar__area--focus': popoverOpen }
+  );
+  const label = i18n._t('ElementAddNewButton.ADD_BLOCK', 'Add block');
```

Comment:
I use the same label as the AddNewButton. We might want to add something more contextual like "Insert block".